app/uploaders/syllabus_uploader.rb

```diff
@@ -17,7 +17,7 @@ class SyllabusUploader < CarrierWave::Uploader::Base
   # Override the directory where uploaded files will be stored.
   # This is a sensible default for uploaders that are meant to be mounted:
   def store_dir
-    "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+    "#{Rails.root.to_s}/uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
```

Comment:
The default behavior in interpolation in ruby essentially does a to_s, so you can just do `"#{Rails.root}/etc"` here.